distribution/docs/src/main/resources/content/_whitelists/catalog-whitelist.adoc

```diff
@@ -6,39 +6,167 @@
+
+In package `ddf.catalog.data`
+
+* `Attribute`
+* `AttributeDescriptor`
+* `AttributeType`
+* `BinaryContent`
+* `ContentType`
+* `Metacard`
+* `MetacardCreationException`
+* `MetacardType`
+* `MetacardTypeUnregistrationException`
+* `Result`
+
+In package `ddf.catalog.event`
+
+* `DeliveryException`
+* `DeliveryMethod`
+* `EventException`
+* `EventProcessor`
+* `InvalidSubscriptionException`
+* `Subscriber`
+* `Subscription`
+* `SubscriptionExistsException`
+* `SubscriptionNotFoundException`
+
+In package `ddf.catalog.federation`
+
+* `Federatable`
+* `FederationException`
+* `FederationStrategy`
+
+In package `ddf.catalog.filter`
+
+* `AttributeBuilder`
+* `BufferedSpatialExpressionBuilder`
+* `ContextualExpressionBuilder`
+* `EqualityExpressionBuilder`
+* `ExpressionBuilder`
+* `FilterAdapter`
+* `FilterBuilder`
+* `FilterDelegate`
+* `NumericalExpressionBuilder`
+* `NumericalRangeExpressionBuilder`
+* `SpatialExpressionBuilder`
+* `TemporalInstantExpressionBuilder`
+* `TemporalRangeExpressionBuilder`
+* `XPathBasicBuilder`
+* `XPathBuilder`
+
+In package `ddf.catalog.operation`
+
+* `CreateRequest`
+* `CreateResponse`
+* `DeleteRequest`
+* `DeleteResponse`
+* `Operation`
+* `OperationTransaction`
+* `Pingable`
+* `ProcessingDetails`
+* `Query`
+* `QueryRequest`
+* `QueryResponse`
+* `Request`
+* `ResourceRequest`
+* `ResourceResponse`
+* `Response`
+* `SourceInfoRequest`
+* `SourceInfoResponse`
+* `SourceProcessingDetails`
+* `SourceResponse`
+* `Update`
+* `UpdateRequest`
+* `UpdateResponse`
+
+In package `ddf.catalog.plugin`
+
+* `AccessPlugin`
+* `PluginExecutionException`
+* `PolicyPlugin`
+* `PolicyResponse`
+* `PostFederatedQueryPlugin`
+* `PostIngestPlugin`
+* `PostQueryPlugin`
+* `PostResourcePlugin`
+* `PreDeliveryPlugin`
+* `PreFederatedQueryPlugin`
+* `PreIngestPlugin`
+* `PreQueryPlugin`
+* `PreResourcePlugin`
+* `PreSubscriptionPlugin`
+* `StopProcessingException`
+
+In package `ddf.catalog.resource`
+
+* `DataUsageLimitExceededException`
+* `Resource`
+* `ResourceNotFoundException`
+* `ResourceNotSupportedException`
+* `ResourceReader`
+* `ResourceWriter`
+
+In package `ddf.catalog.service`
+
+* `ConfiguredService`
+
+In package `ddf.catalog.source`
+
+* `CatalogProvider`
+* `ConnectedSource`
+* `FederatedSource`
+* `IngestException`
+* `InternalIngestException`
+* `RemoteSource`
+* `Source`
+* `SourceDescriptor`
+* `SourceMonitor`
+* `SourceUnavailableException`
+* `UnsupportedQueryException`
+
+In package `ddf.catalog.transform`
+
+* `CatalogTransformerException`
+* `InputCollectionTransformer`
+* `InputTransformer`
+* `MetacardTransformer`
+* `QueryResponseTransformer`
+
+In package `ddf.catalog.transformer.api`
+
+* `MetacardMarshaller`
+* `PrintWriter`
+* `PrintWriterProvider`
+
+In package `ddf.catalog.util`
+
+* `Describable`
+* `Maskable`
+
+In package `ddf.catalog.validation`
+
+* `MetacardValidator`
+* `ValidationException`
+
+In package `org.codice.ddf.endpoints`
+
+* `OpenSearch` Generated class????
```

Comment:
Really don't think this should be included

Reply:
I don't know why any of the endpoints would be in the whitelist. This looks like a completely mistake to me.

Reply:
removed